### ISSUE TITLE
Getting higher COCO accuracy?

### ISSUE THREAD

**Issue by kentaroy47** [jwyang/faster-rcnn.pytorch#359]
The benchmark result provided in Detectron's readme is high (37.1% COCOmAP) for Resnet50, even though with faster-rcnn.
https://github.com/roytseng-tw/Detectron.pytorch/blob/master/BENCHMARK.md

Do anyone know where the accuracy difference come from? (e.g. dataset selection, learn rate tweaking..?)

**Comment by jwyang** [comment]
@kentaroy47 the training and validation set is different. This might explain the differences.

**Comment by kentaroy47** [comment]
@jwyang Thanks. Yes, noticed that they use COCO2017 train/val set.
I guess they have more data than initial COCO, concluding in a higher accuracy.

Good to know that there are no fundamental differences.
Will close the issue.
